packages/server/test/index.test.tsx

```diff
@@ -29,34 +29,14 @@ test('mix query and mutation', async () => {
       },
     });
 
-  expect(
-    await r.invoke({
-      target: 'queries',
-      path: 'q1',
-      input: null,
-      ctx: {},
-    }),
-  ).toMatchInlineSnapshot(`"q1res"`);
+  const caller = r.createCaller({});
+  expect(await caller.query('q1')).toMatchInlineSnapshot(`"q1res"`);
 
-  expect(
-    await r.invoke({
-      target: 'queries',
-      path: 'q2',
-      input: {
-        q2: 'hey',
-      },
-      ctx: {},
-    }),
-  ).toMatchInlineSnapshot(`"q2res"`);
+  expect(await caller.query('q2', { q2: 'hey' })).toMatchInlineSnapshot(
+    `"q2res"`,
```

Comment:
should we not test the inputs getting sent through properly? Change the router above from
```ts
    .query('q2', {
      input: z.object({ q2: z.string() }),
      resolve() {
        return 'q2res';
      },
    })
```
to
```ts
    .query('q2', {
      input: z.object({ q2: z.string() }),
      resolve({ input }) {
        return `q2res: ${input.q2}`;
      },
    })
```

Then
```suggestion
    `"q2res: hey"`,
```